subprojects/docs/src/samples/userguide/artifacts/componentModuleMetadata/build.gradle

```diff
@@ -4,8 +4,8 @@ apply plugin: 'java'
 dependencies {
     modules {
         module("com.google.collections:google-collections") {
-            replacedBy("com.google.guava:guava")
+            replacedBy("com.google.guava:guava", "Guava replaced google-collections")
```

Comment:
I'd like to see a better reason here. Maybe something like "Google decided to rename and relocate Google Collections". As a user I might also be interested in a link to additional resources. Maybe something you could discuss as future improvement.

Reply:
I don't want to make the text too long in the example. That's why I kept it vague. I reworded it a bit but kept it short. It's not only a relocation, as Guava includes not only collections but also other libraries

In general you can add a longer reason (e.g., include a link).